### PR TITLE
Issue244 document buttons functionality

### PR DESCRIPTION
**Pull request by xquetzalcoatl**
Documented functionality to include buttons

### CODE REVIEW

**Comment by geoperez** [comment]
Please resolve conflicts.

File: srcdocs/pages/Sample.tsx [hunk 1]
```diff
@@ -87,14 +87,14 @@ export default withStyles(styles)(
           <Grid container={true} spacing={24} className={classes.container}>
             <Grid item={true} xs={12} className={classes.paper}>
               <Paper className={classes.paper}>
-                <Typography variant='h4'>Samples</Typography>
+                <Typography variant='display1'>Samples</Typography>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

File: srcdocs/pages/Sample.tsx [hunk 2]
```diff
@@ -87,14 +87,14 @@ export default withStyles(styles)(
...
                 <Divider />
                 <br />
                 <Grid item={true} xs={12} className={classes.paper}>
-                  <Typography variant='h5' gutterBottom={true}>
+                  <Typography variant='headline' gutterBottom={true}>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

File: srcdocs/pages/Sample.tsx [hunk 3]
```diff
@@ -87,14 +87,14 @@ export default withStyles(styles)(
...
                     Basic grid features
                   </Typography>
-                  <Typography variant='h6'>
+                  <Typography variant='subheading'>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

File: srcdocs/pages/Sample.tsx [hunk 4]
```diff
@@ -142,10 +142,10 @@ export default withStyles(styles)(
                 </Grid>
                 <Divider />
                 <Grid item={true} xs={12} className={classes.paper}>
-                  <Typography variant='h5' gutterBottom={true}>
+                  <Typography variant='headline' gutterBottom={true}>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

File: srcdocs/pages/Sample.tsx [hunk 5]
```diff
@@ -142,10 +142,10 @@ export default withStyles(styles)(
...
                     Full grid features
                   </Typography>
-                  <Typography variant='h6'>
+                  <Typography variant='subheading'>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

File: srcdocs/pages/Sample.tsx [hunk 6]
```diff
@@ -257,7 +257,7 @@ export default withStyles(styles)(
                   <Typography variant='headline' gutterBottom={true}>
                     Give another presentation to your data.
                   </Typography>
-                  <Typography variant='h6'>
+                  <Typography variant='subheading'>
```
Comment (AnyxLeo) [inline srcdocs/pages/Sample.tsx]: Rever this merge the correct variant syntaxis is the previous one

**Pull request merged**